packages/devtools/src/pages/project/config-editor/index.tsx

```diff
@@ -15,7 +15,7 @@ import "ace-builds/src-noconflict/ext-language_tools";
 import "./style.scss";
 import { useApp, useLocale, useProject } from "../../../models/app.model";
 import { useModel } from "@bramblex/state-model-react";
-import { tryOrAlert } from "../../../common/utils";
+import { tryOrAlert, classNames } from "../../../common/utils";
```

Comment:
It's has already imported `classnames` package, so there is no need to implement it again.

Reply:
What is the principle of using third-party libraries? I thought the rule was to only use them if necessary (otherwise, why not use Redux?).